src/main/features/response/draft/responseDraft.ts

```diff
@@ -187,13 +187,19 @@ export class ResponseDraft extends DraftDocument {
   }
 
   public isResponseRejected (): boolean {
-    if (!this.isResponsePopulated()) {
+    if (!this.response || !this.response.type) {
```

Comment:
isn't this exactly what `isResponsePopulated` does?

Reply:
I thought so too, but changing it fixed a test... Will revisit